File: app/src/main/java/com/example/padamlight/data/local/Suggestion.java
package com.example.padamlight.data.local;

import android.support.annotation.DrawableRes;

import com.example.padamlight.R;
import com.google.android.gms.maps.model.LatLng;

/**
 * Suggestion model
 * Defines suggestion in From or To textfields
 */
public class Suggestion {
    private LatLng latLng;

    @DrawableRes
    private int icon;

    public Suggestion(LatLng latLng) {
        this.latLng = latLng;
        this.icon = R.drawable.ic_favorite;
    }

    public LatLng getLatLng() {
        return latLng;
    }

    @DrawableRes
    public int getIcon() {
        return icon;
    }

    public String toString(){
        return latLng.latitude + "," + latLng.longitude;
    }
}

File: app/src/main/java/com/example/padamlight/data/local/Suggestion.kt
package com.example.padamlight.data.local

import android.support.annotation.DrawableRes

import com.example.padamlight.R
import com.google.android.gms.maps.model.LatLng

/**
 * Suggestion model
 * Defines suggestion in From or To textfields
 */
data class Suggestion(val latLng: LatLng) {

    override fun toString(): String {
        return latLng.latitude.toString() + "," + latLng.longitude
    }
}
